### ISSUE TITLE
Cannot load templates: {}

### ISSUE THREAD

**Issue by lucadalli** [wxt-dev/wxt#528]
### Describe the bug

Facing `Cannot load templates: {}` error on Windows 11 machine
Tried resetting modem to change external IP, didn't help,

### To Reproduce

`pnpm dlx wxt@latest init wxt-extension`
 or
`npx wxt@latest init wxt-extension`

Same error both in monorepo and outside. Tried may different versions, still same error.

### Expected behavior

Templates load so that one can be selected.

### Environment

```
  System:
    OS: Windows 11 10.0.22621
    CPU: <redacted>
    Memory: 5.27 GB / 15.87 GB
  Binaries:
    Node: 16.20.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - C:\Program Files\nodejs\yarn.CMD
    npm: 8.19.4 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.15.4 - ~\AppData\Local\pnpm\pnpm.EXE
  Browsers:
    Edge: Chromium (122.0.2365.66)
    Internet Explorer: 11.0.22621.1
```

**Comment by aklinker1** [comment]
I've added better error logs, let me know if you still face this error and share the output of `curl -v https://ungh.cc/repos/wxt-dev/wxt/files/main`.

**Comment by lucadalli** [comment]
Error persists. You haven't released the change with the better logs so error is still `Cannot load templates: {}`

Output of `curl -v https://ungh.cc/repos/wxt-dev/wxt/files/main`:
```
VERBOSE: GET with 0-byte payload
VERBOSE: received -1-byte response of content type application/json


StatusCode        : 200
StatusDescription : OK
Content           : {"meta":{"sha":"38345481dfbd65ec6e06f27bcd26397287db18cf"},"files":[{"path":".codecov.yml","mode":"
                    100644","sha":"bfdc9877d9acc165c0342befa33f2e8f769538cf","size":130},{"path":".gitattributes","mode
                    ":...
RawContent        : HTTP/1.1 200 OK
                    Transfer-Encoding: chunked
                    Connection: keep-alive
                    Access-Control-Allow-Origin: *
                    access-control-allow-headers: *
                    access-control-allow-methods: *
                    access-control-max-age: 0
                    Cf-Pla...
Forms             : {}
Headers           : {[Transfer-Encoding, chunked], [Connection, keep-alive], [Access-Control-Allow-Origin, *],
                    [access-control-allow-headers, *]...}
Images            : {}
InputFields       : {}
Links             : {}
ParsedHtml        : System.__ComObject
RawContentLength  : 46097
```

**Comment by lucadalli** [comment]
I was on node v16.20.1 as I was doing some work on legacy projects.
Switched to node v18.16.1 and no longer facing error.

**Comment by aklinker1** [comment]
Released better logging in [v0.17.8](https://github.com/wxt-dev/wxt/releases/tag/v0.17.8)
